### ISSUE TITLE
10.0 Dragonflight PTR

### ISSUE THREAD

**Issue by yoshimo** [millanzarreta/ClassicUI#1]
complete UI revamp on the side of Blizzard so might require some work

`1x Interface/AddOns/ClassicUI/ClassicUI.lua:1305: attempt to index global 'MainMenuBarArtFrameBackground' (a nil value)
[string "@Interface/AddOns/ClassicUI/ClassicUI.lua"]:1305: in function `MainFunction'
[string "@Interface/AddOns/ClassicUI/ClassicUI.lua"]:244: in function <Interface/AddOns/ClassicUI/ClassicUI.lua:223>
[string "=[C]"]: ?
[string "@Interface/AddOns/Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:66: in function <Interface/AddOns/Ace3/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:494: in function `InitializeAddon'
[string "@Interface/AddOns/Ace3/AceAddon-3.0-13/AceAddon-3.0.lua"]:619: in function <Interface/AddOns/Ace3/AceAddon-3.0/AceAddon-3.0.lua:611>`

**Comment by millanzarreta** [comment]
In Dragonflight the interface has been completely changed. Such a big change makes the ClassicUI addon stop working, as a near-scratch rewrite is needed to bring the old interface back.

Some of the extra ClassicUI options still seem to work, but others need to be rewritten as well.

My intention is to keep this addon active in Dragonflight, but due to such a big interface change, it needs a complete rewrite which will take a long time. The update will come one day, but it all depends on the free time you have available to develop it and how complicated it is.

In short: I'm working on the update, but it may take a long time...

**Comment by AadaEa** [comment]
Hi, will you also deal with the character frames (which includes the gold and silver dragons around the portrait) ?
The new char frames has a different vibe, similar to the new minimalist modern ui.

The "empty" icon inside the bags changed too, and the bags feel different in a way I can't explain, it feels narrower.

There's also some issues when moving the wow window from one monitor to another when the monitor has a different size/aspect ratio. Same kind of issue when you jump from windowed to full screen and vice versa.


Thank you for your work, I'll try to donate and let other know about this addon.
Please take the time you need, don't feel rushed, the new UI can be made (somewhat) playable for a time, what's missing is mostly the classic vibe.

**Comment by millanzarreta** [comment]
Updated to Dragonflight (ClassicUI v2.0.0!)

The restoration of some frames by their classic versions is planned to be included in future versions. This is the list of frames that are not yet supported and that are planned to be included in the future (although the list could change and include some more):
- PlayerFrame, TargetFrame, FocusFrame
- Castbar
- Lootframe
